css/styles.css

```diff
@@ -7,16 +7,50 @@
   --brand-color4: rgba(244, 244, 253, 1);
   --brand-color5: rgba(255, 255, 255, 1);
   --brand-color6: rgba(64, 75, 191, 1);
+  --brand-color7: rgba(231, 233, 252, 1);
```

Comment:
It is better to name variable as actual color like "--brand-red7" rather than " --brand-color7"